internal/response-consumer/handler.go

```diff
@@ -80,6 +80,8 @@ func (this *handler) onMessage(ctx context.Context, msg *k.Message) {
 			status = inferSatStatus(value.SatEvents, nil)
 			eventsSerialized = utils.MustMarshal(value.SatEvents)
 
+			satellite.SortSatEvents(value.SatEvents)
```

Comment:
if you do the sorting earlier then other pieces of code can benefit from events being ordered (e.g. the events stored in the database will be sorted also)